pkg/controller/controlplane/valuesprovider.go

```diff
@@ -301,6 +301,11 @@ func getCCMChartValues(
 
 	if cpConfig.CloudControllerManager != nil {
 		values["featureGates"] = cpConfig.CloudControllerManager.FeatureGates
+		if cpConfig.CloudControllerManager.CloudControllerNetworking != nil {
+			values["networking"] = map[string]any{
```

Comment:
Can we define the key values as constants.

Reply:
good point, done